### ISSUE TITLE
Add "Switch body encoding"

### ISSUE THREAD

**Issue by sw33tLie** [caido/caido#99]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Description

A button to switch the body encoding of a POST request would be handy.
By that, I mean something able to convert a standard POST body to a `multipart/form-data` one.
Doing it manually is a huge pain so it's really needed.

Thanks!

**Comment by Corb3nik** [comment]
Yeah that's a nice feature to have. Conversions between the following would be 👌 :
- application/x-www-form-urlencoded
- multipart/form-data
- JSON
- XML

**Comment by sw33tLie** [comment]
Oh yes, absolutely.
JSON => XML would be dope for testing XXE also!

**Comment by Sytten** [comment]
We should probably emulate some features of postman for that. Ironically this is very close to the tooling that dev use.
